src/library/ksPie/ksPie.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import KsLegend from '../ksLegend/ksLegend';
+import { KuiPie } from '@kleeen/visualizations-react';
+
+import './ksPie.scss';
+
+const KsPie = ({ context }) => {
```

Comment:
You do not need to do this, but ideally we should be able to update this from the outside world.
```suggestion
const KsPie = ({ context, height, width }) => {
```

Reply:
Done 👍